### DISCUSSION TITLE
cart systemd error , ansible setup

### DISCUSSION THREAD

**Discussion by priyanshuprafful** [learndevopsonline/learndevopsonline#1509] [Q&A]
Getting this error , not able to resolve it 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/27334699/63261932-2134-43fc-81c1-5f920afeccc8)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/27334699/2ccf432c-e511-496f-8c6a-d786a3ff8dc0)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/27334699/41754402-abf5-4f79-8ad8-76b43d394c3f)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/27334699/448b105e-d910-45c0-ad68-577f5bfa442d)

**Comment by priyanshuprafful** [comment]
@r-devops , unable to solve this , tried multiple times on different machines , t3.small , t3.micro etc .

**Reply by sreedharm07** [reply]
Hi @priyanshuprafful  can you please turn on your machine

**Reply by priyanshuprafful** [reply]
hey @sreedharm07 , I have terminated the machine and actually I have copied files/cart.service under tasks , that was the problem , I have resolved it though and it is working fine now . Thanks anyway though 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/27334699/0dc1065c-0749-4edd-855b-ce6acb3d6854)

**Reply by sreedharm07** [reply]
ok  thank you bro
